kivy/core/window/__init__.py

```diff
@@ -211,6 +211,10 @@ class WindowBase(EventDispatcher):
             Width of the window.
         `height`: int
             Height of the window.
+        `minimum_width`: int
+            Minimum width of the window
+        `minimum_height`: int
+            Minimum height of the window
```

Comment:
It need a `.. versionadded:: 1.9.1` tag, check in the docstring, there should be one for other added token as well.